script.js

```diff
@@ -0,0 +1,86 @@
+//task 1
+var firstTask = document.getElementById('firstTask');
+var innerHTML = '<p>Все простые числа от 0 до 100: ';
+var result = [];
+var i = 2;
+while (i <= 100) {
+
+    if (isNumberPrime(i) && result.indexOf(i) === -1) { //Избавляемся от дублей в результате

```

Comment:
так будет более читаемо и понятно :)
```suggestion
    if (isNumberPrime(i) && !result.includes(i)) { //Избавляемся от дублей в результате
```